packages/masa-source/src/services/twitter/index.ts

```
@@ -0,0 +1,53 @@
+    ...(baseQuery &&
+      !(
+        platformSpecificOptions.platformArgs as Partial<TwitterQueryOptionsInput>
+      )?.allWords && { allWords: baseQuery }),
+    ...(pageSize &&
+      !(
+        platformSpecificOptions.platformArgs as Partial<TwitterQueryOptionsInput>
+      )?.pageSize && { pageSize: pageSize }),
+  };
+
+  if (rawOptions.pageSize === undefined) {
+    delete rawOptions.pageSize;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)